src/dotnet/APIView/APIViewWeb/Repositories/BlobCodeFileRepository.cs

```diff
@@ -67,7 +67,12 @@ public async Task<CodeFile> GetCodeFileWithCompressionAsync(string revisionId, s
             }
             var info = await client.DownloadAsync();
             codeFile = await CodeFile.DeserializeAsync(info.Value.Content, doTreeStyleParserDeserialization: true);
-            if (updateCache)
+
+            if (invalidateCache)
+            { 
+                _cache.Remove(key);
```

Comment:
I don't think this will be enough. Call will return at line number 66 even if invalidateCache is true which will return what's in cache (mostly already altered change by previous diff). Let's disable the cache and enable it later once we figure out the token logic.